timesketch/api/v1/resources/upload.py

```diff
@@ -40,9 +41,75 @@
 class UploadFileResource(resources.ResourceMixin, Resource):
     """Resource that processes uploaded files."""
 
+    def _get_index(
+            self, name, description, sketch, index_name='',
+            label='', extension=''):
+        """Returns a SearchIndex object to be used for uploads.
+
+        Args:
+            name: the name of the searchindex.
+            description: the description of the searchindex.
+            sketch: sketch object (instance of Sketch).
+            index_name: optional index name, if supplied and if it exists
+                then the index associated with that will be returned.
+            label: optional label of the data, if supplied will be used to
+                determine whether an already existing index can be
+                used or a new one created.
+            extension: optional file extension if a file is being uploaded,
+                if supplied and no label used, then the extension will be
+                used as a label.
+
+        Returns:
+            A SearchIndex object.
+        """
+        if index_name:
+            if not isinstance(index_name, six.text_type):
```

Comment:
Do we still need to use six? We are python3 only

Reply:
no... this is old remnants and has no place here anymore ;)